docs/src/main/paradox/features/version-warning.md

```diff
@@ -0,0 +1,12 @@
+Version warning
+---------------
+
+Paradox supports showing a warning when users browse documentation which is not the current released version. This is achieved by storing a JSON file (`paradox.json`) together with the generated site and specifying a stable URL to where the released version will be available.
+
+The built in theme (`generic`) contains Javascript to fetch the JSON file and compare the version with the version for which the documentation showing was generated. Whenever they differ, a warning text shows on every page offering a link to the released version's page.
+
+To use this functionality, add `project.url` to Paradox properties
+
+```scala
+paradoxProperties += ("project.url" -> "https://developer.lightbend.com/docs/paradox/current/")
```

Comment:
I see that the URL that is used for the `paradox.json` contains a double slash:

> The Same Origin Policy disallows reading the remote resource at https://doc.akka.io/docs/akka-stream-kafka/current//paradox.json

Reply:
Yes, I wanted it to work in case the `project.url` doesn't end with a slash.